src/fonts.css

```diff
@@ -0,0 +1,18 @@
+@font-face {
+  font-family: "Noto Sans Condensed";
+  src: url("https://osm-americana.github.io/web-fonts/noto/NotoSans-CondensedMedium-PropNums.woff2")
+      format("woff2"),
+    url("https://osm-americana.github.io/web-fonts/noto/NotoSans-CondensedMedium-PropNums.woff")
+      format("woff");
+  font-weight: 500;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Noto Sans Armenian Condensed";
+  src: url("https://github.com/osm-americana/web-fonts/raw/main/noto/NotoSansArmenian-CondensedMedium-PropNums.woff2")
```

Comment:
Rather than pointing to the raw file, we should deploy to github.io and point to that location.

Reply:
Shouldn't the font files be pulled in by the build script so they are served from the same domain as the rest of the style?